specs/assets_overview/03_technical_details.rst

```diff
@@ -0,0 +1,21 @@
+=========================
+Technical Details
+=========================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`64`_  Assets overview  tschubotz  2019-01-22
+=====  ===============  =========  ==========
+
+.. _64: https://github.com/gnosis/safe/issues/64
+
+
+#. `Main`_
+#. `User interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
```

Comment:
Here we need to add details mentioning:

- how to get the balance of ETH account
- how to get the balance of ERC20 account
- how to get the list of enabled tokens
  - should it be local (current implementation) (user preference stored in the app) or should be on the server? (then endpoint for enabling / disabling token should be added). Currently, the iOS treats "default" property of a token from the `tokens` endpoint as a flag to show or hide that token by default. New `default` tokens not seen by the app before being added to the tokens list overview automatically.
- Maybe reference the `token list` spec here?

Reply:
Since this is technical details, someone of you would need to take this over, please. @rmeissner @DmitryBespalov @fmrsabino